### DISCUSSION TITLE
In my app, I used react-native-gesture-handler/Swipeable,but iOS's default swipe right action to return to the previous page does not take effect

### DISCUSSION THREAD

**Discussion by devoutDisciple** [software-mansion/react-native-gesture-handler#2508] [Q&A]
iOS's default swipe right action to return to the previous page does not take effect，Excuse me, how do you solve it?

**Comment by m-bert** [comment]
Hi! Could you check if this issue persists with [ReanimatedSwipeble](https://docs.swmansion.com/react-native-gesture-handler/docs/components/reanimated_swipeable/)?

**Comment by K-E-L** [comment]
I'm getting the same problem, and am using "import Swipeable from 'react-native-gesture-handler/ReanimatedSwipeable' "
